python/paddle/distributed/auto_parallel/operators/embedding.py

```diff
@@ -0,0 +1,97 @@
+# Copyright (c) 2021 PaddlePaddle Authors. All Rights Reserved.
```

Comment:
"auto_parallel/operators/embedding.py"  rename to "auto_parallel/operators/dist_embedding.py" ?

Reply:
Done!